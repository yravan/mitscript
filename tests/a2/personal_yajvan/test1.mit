
fibonacci = fun(n){
                if(n == 0 | n == 1){return 1;}
                return fibonacci(n -1) + fibonacci(n - 2);
            };

n1 = 10;
n2 = 11;
print("F11 + F10: " + (fibonacci(n1) + fibonacci(n2)));
print("F11 + F10 - F12: " + (fibonacci(n1) + fibonacci(n2) - fibonacci(12)));
