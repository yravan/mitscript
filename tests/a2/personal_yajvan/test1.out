F11 + F10: 233
F11 + F10 - F12: 0
